lib/osut/utils.rb

```diff
@@ -3099,7 +3099,7 @@ def getNonCollinears(pts = nil, n = 0)
       p3  = pts[i3]
       v13 = p3 - p1
       v12 = p2 - p1
-      next if v12.cross(v13).length < TOL
+      next if v12.cross(v13).length < TOL2
```

Comment:
This is one of maybe several tweaks to ensure OSut doesn't red-flag surfaces that EnergyPlus would ultimately tolerate. The latter would accept/process surfaces having a width/length of my thumb (definitely not good, but hey). More to come, maybe.